client/src/app/Root.jsx

```diff
@@ -27,7 +34,7 @@ const Root = () => {
       }
     } catch (error) {
       setAuthorizeData({
-        isAuthorized: false,
+        isAuthorized: true,
```

Comment:
We cannot authorize the app without a valid token. In case of an error, the authorization is revoked
```suggestion
        isAuthorized: false,
```